content/applications/finance/expenses/log_expenses.rst

```diff
@@ -0,0 +1,240 @@
+- :guilabel:`Paid By`: click the radio button to indicate who paid for the expense, and should be
+  reimbursed. If the employee paid for the expense and should be reimbursed, select
+  :guilabel:`Employee (to reimburse)`. If the company paid directly instead (e.g. if the company
+  credit card was used to pay for the expense) select :guilabel:`Company`. Depending on the expense
+  category selected, this field may not appear.
+- :guilabel:`Expense Date`: using the calendar module, enter the date the expense was incurred. Use
+  the :icon:`fa-chevron-left` :guilabel:`(left)` and :icon:`fa-chevron-right` :guilabel:`(right)`
+  arrows to navigate to the correct month, then click on the specific day to enter the selection.
+- :guilabel:`Account`: select the expense account that the expense should be logged in from the
+  drop-down menu.
+- :guilabel:`Customer to Reinvoice`: if the expense is something that should be paid for by a
+  customer, select the :abbr:`SO (Sales Order)` and customer that should be invoiced for this
+  expense from the drop-down menu. All sales orders in the drop-down menu list both the :abbr:`SO
+  (Sales Order)` as well as the company the sales order is written for. After the expense is saved,
+  the customer name disappears and only the :abbr:`SO (Sales Order)` is visible on the expense.
+
+  .. example::
+     A customer wishes to have an on-site meeting for the design and installation of a custom
+     garden, and agrees to pay for the expenses associated with it, such as travel, hotel, meals,
+     etc. All expenses tied to that meeting would indicate the sales order for the custom garden
+     (which also references the customer) as the :guilabel:`Customer to Reinvoice`.
+
+- :guilabel:`Analytic Distribution`: select the account the expense should be written against from
+  the drop-down menu for either :guilabel:`Projects`, :guilabel:`Departments`, or both. Multiple
+  accounts can be listed for each category, if needed. Adjust the percentage for each analytic
+  account by typing in the percentage value next to each account.
+- :guilabel:`Company`: if multiple companies are set up, select the company the expense should be
+  filed for from the drop-down menu. The current company automatically populates this field.
+- :guilabel:`Notes...`: if any notes are needed to clarify the expense, enter them in the notes
+  field.
+
+.. image:: log_expenses/expense-filled-in.png
+   :align: center
+   :alt: A filled in expense form for a client lunch.
+
+Attach receipts
+---------------
+
+After the expense record is created, the next step is to attach a receipt. Click the
+:guilabel:`Attach Receipt` button, and a file explorer appears. Navigate to the receipt to be
+attached, and click :guilabel:`Open`.
+
+The new receipt is recorded in the chatter, and the number of receipts appears next to the
+:icon:`fa-paperclip` :guilabel:`(paperclip)` icon beneath the expense form. More than one receipt
```

Comment:
depending on the browser size, like we've discussed in the past, the chatter might be to the side *or* the bottom. So, make sure to keep things as universal/generic as possible when describing elements such as this. Thanks

Reply:
I keep forgetting that- I will make sure to be more clear (by being more vague ^_^)